audit/projects/k8s-staging-capi-docker/services/enabled.txt

```diff
@@ -1,11 +1,13 @@
 NAME                              TITLE
 cloudbuild.googleapis.com         Cloud Build API
 cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
+compute.googleapis.com            Compute Engine API
```

Comment:
more APIs getting enabled that shouldn't be, I'm guessing this is one of the org admins migrating around on console...